service/history/historyBuilder.go

```diff
@@ -29,15 +29,6 @@ import (
 	"github.com/uber/cadence/common/persistence"
 )
 
-const (
-	emptyVersion int64 = -2234
```

Comment:
looks like we have 2 different values here, @wxing1292 , which value should we use? -2234 or -24.

Reply:
does not matter, since this value is only used for cross dc, and cross dc is not enabled yet